internal/core/metadata/operators/command/mocks/CommandsByNameExecutor.go

```diff
@@ -0,0 +1,34 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
```

Comment:
Do we need a mock for the Executor?  I feel like this along with the other two mock Executor files can be removed from /operators/command/mocks/

Reply:
No, we don't . Removed